Topics/User_Experience/navigation design in complex websites.md

```diff
@@ -0,0 +1,64 @@
+## What is Navigation Design ?
+Navigation Design, also referred to as menu design, helps users to easily move through the pages of your website. It utilizes design techniques to allow ease of navigation and provide the user a better sure experience. 
+
+
+## Good vs Bad Navigation Design 
+Good navigation design allows for ease of  finding information, is intuitive and easy to use. On the other hand, bad navigation design can overwhelm the user and make finding important content difficult.  As a result,  it is important to keep the user in mind when designing the navigation of a website, especially complex websites with a multitude of pages. Reasoning about how the user can navigate through the many pages of a website, can help lay the foundation to good navigation design. 
+
+Consider using the following steps to start thinking of good navigation design  : 
+
+1. Layout all the pages on your website whether on paper of through a design tool such as Figma 
```

Comment:
Great tips! I think it would also be helpful to incorporate some examples of other forms of navigation design, since it not only encompasses finding your way between pages, but also figuring out how to use the features on a single page as well. For example, if there's a search bar and the user can apply filters, they should be able to easily find the button to apply the filters and be aware that the filters are being applied. Making it easy for the user to know their way around a single page and applying familiar format designs to accomplish this are also necessary parts of navigation design!

Reply:
Great point ! I'll be sure to include some resources on searching within a page and add this point to the things to consider section as well !